### PR TITLE
Added English translation for README and fixed typos in Russian

### PR DESCRIPTION
**Pull request by dariakameneva**
There's no guide for contributors, but I hope it's fine :)

A fix for #1 

- Added English translation for README
- Fixed some typos in Russian version
- Added a link from README.md (which is in Russian) to the English version

### CODE REVIEW

**Comment by luza** [comment]
Brilliant! Thank you!

**Comment by luza** [comment]
Btw, feel free to put English readme in place of Russian (Russian could be moved to README-ru.md).

File: README-en.md [hunk 1]
```diff
@@ -0,0 +1,855 @@
...
+
+    - requestArgs:
+        orderNr: ORDER0002
+      responseArgs:
+        200:
+          userId: '0001'
+          amount: 72000
+```
+
+### HTTP-запрос
```
Comment (luza) [inline README-en.md]: Piece of Russian.

File: README-en.md [hunk 2]
```diff
@@ -0,0 +1,855 @@
...
+        scriptArgs:
+          file_name: "cmd_recalculate_customer_1.sh"
+```
+
+### A DB request
+
+After HTTP request execution you can run an SQL request to DB to check the data changes.
+The response can contain several records. Those records are compared to the expected list of records.
+
+#### Request definition
```
[the remark this answers is not in the document]
Comment (dariakameneva) [reply README-en.md]: 🤦‍♀ makes sense

**Comment by dariakameneva** [comment]
Thank you for the review! I don't have the rights to merge it :)

**Pull request merged**